dll/win32/msgina/resource.h

```diff
@@ -122,3 +122,9 @@
 #define IDS_SHUTDOWN_RESTART_DESC     50007
 #define IDS_SHUTDOWN_SLEEP_DESC       50008
 #define IDS_SHUTDOWN_HIBERNATE_DESC   50009
+
+/* Emergency Restart */
+#define IDS_EMERGENCY_RESTART_TITLE   50010
+#define IDS_EMERGENCY_RESTART         50011
+#define IDS_FORCE_LOGOFF_TITLE        50012
+#define IDS_FORCE_LOGOFF              50013
```

Comment:
What about having the logoff stuff declared/defined first before the shutdown/restart? (as done in other places there)